mitodl/micromasters
# Populate ExamProfile timestamp values

#### What are the relevant tickets?
Second part of #3025 

#### What's this PR do?
Builds on  #3083 - this PR populates data into the `updated_on` and `created_on` fields of `ExamProfile`, sets those fields as no longer nullable, and updates the profile writer to use the new `updated_on` field.

#### How should this be manually tested?
Create an `ExamProfile` record prior to running the migrations. Run the migrations and ensure you get values and the aforementioned fields are not nullable. Run the migrations backwards too.
